<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandsEditor</class>
 <widget class="QDialog" name="CommandsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commands Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonCreate">
       <property name="text">
        <string>Create</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/resources/images/ic-add.png</normaloff>:/resources/images/ic-add.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonEdit">
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/resources/images/ic-write.png</normaloff>:/resources/images/ic-write.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExecute">
       <property name="text">
        <string>Execute</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/resources/images/ic-play.png</normaloff>:/resources/images/ic-play.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonEnableDisable">
       <property name="text">
        <string>Enabled</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/resources/images/ic-switch-on.png</normaloff>:/resources/images/ic-switch-on.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDelete">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/resources/images/ic-trash.png</normaloff>:/resources/images/ic-trash.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidgetCommands">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
